src/main/java/xyz/groundx/caver_ext_kas/wallet/KASWallet.java

```diff
@@ -0,0 +1,353 @@
+        feeDelegatedTransaction.appendFeePayerSignatures(tx.getFeePayerSignatures());
+
+        return feeDelegatedTransaction;
+    }
+
+    /**
+     * Sign a transaction as a fee payer using an account in KAS Wallet service.
+     * @param feePayerAddress An address of account to sign as a fee payer in KAS Wallet service
+     * @param feeDelegatedTransaction A fee delegated transaction instance.
+     * @return AbstractFeeDelegatedTransaction
+     * @throws ApiException
+     * @throws IOException
+     */
+    @Override
+    public AbstractFeeDelegatedTransaction signAsFeePayer(String feePayerAddress, AbstractFeeDelegatedTransaction feeDelegatedTransaction) throws ApiException, IOException {
+        if(feePayerAddress == null) {
+            return signAsGlobalFeePayer(feeDelegatedTransaction);
+        }
+
+        if(feeDelegatedTransaction.getFeePayer().equals("0x")) {
+            feeDelegatedTransaction.setFeePayer(feePayerAddress);
+        }
+
+        if(!feeDelegatedTransaction.getFeePayer().toLowerCase().equals(feePayerAddress.toLowerCase())) {
+            throw new IllegalArgumentException("Fee payer address are not matched");
+        }
+
+        if(isWeightedMultiSigType(feeDelegatedTransaction, feePayerAddress)) {
+            throw new IllegalArgumentException("Not supported: Using multiple keys in an account is currently not supported.");
+        }
+
+        feeDelegatedTransaction.fillTransaction();
+        String rlp = feeDelegatedTransaction.getRLPEncoding();
+
+        FDUserProcessRLPRequest rlpRequest = new FDUserProcessRLPRequest();
+        rlpRequest.setRlp(rlp);
+        rlpRequest.setFeePayer(feeDelegatedTransaction.getFeePayer());
+        rlpRequest.setSubmit(false);
+
+        if(feeDelegatedTransaction instanceof AbstractFeeDelegatedWithRatioTransaction) {
+            rlpRequest.setFeeRatio(((AbstractFeeDelegatedWithRatioTransaction) feeDelegatedTransaction).getFeeRatioInteger().longValue());
+        }
+
+        FDTransactionResult result = this.walletAPI.requestFDRawTransactionPaidByUser(rlpRequest);
+
+        AbstractFeeDelegatedTransaction tx = (AbstractFeeDelegatedTransaction)TransactionDecoder.decode(result.getRlp());
+        feeDelegatedTransaction.appendFeePayerSignatures(tx.getFeePayerSignatures());
+
+        return feeDelegatedTransaction;
+    }
+
+    /**
+     * Getter function for walletAPI
+     * @return Wallet
+     */
+    public Wallet getWalletAPI() {
+        return walletAPI;
+    }
+
+    /**
+     * Setter function for walletAPI
+     * @param walletAPI The WalletAPI instance to use KAS Wallet API.
+     */
+    public void setWalletAPI(Wallet walletAPI) {
+        this.walletAPI = walletAPI;
+    }
+
+
+    private boolean isWeightedMultiSigType(AbstractTransaction transaction, String address) throws IOException {
+        AccountKey res = transaction.getKlaytnCall().getAccountKey(address).send();
+        if(res == null) {
+            return true;
```

Comment:
```suggestion
            return false;
```
Is this true? not false?